test/unit/bvlc.spec.js

```diff
@@ -13,7 +13,7 @@ describe('bacstack - BVLC layer', () => {
       len: 4,
       func: 10,
       msgLength: 1482,
-      forwardedFrom: null,
+      originatingIP: null,
```

Comment:
I found it confusing when you are generating a new packet and you have to specify `forwardedFrom` as your own (device) IP address, when you aren't the one forwarding it (the BBMD is).

If you are writing a BBMD, you find the sender's IP in the `forwardedFrom` when you haven't forwarded it yet, which is also confusing to me.

I still use `forwardedFrom` where it makes sense, in `client.js` when you have received a packet from a BBMD.

If you have any other suggestions for a name let me know, but anything with "forwarded" in it can be misleading in my opinion.